src/main/java/g0301_0400/s0352_data_stream_as_disjoint_intervals/readme.md

```diff
@@ -16,7 +16,20 @@ Implement the `SummaryRanges` class:
 
 **Output:** \[null, null, \[\[1, 1\]\], null, \[\[1, 1\], \[3, 3\]\], null, \[\[1, 1\], \[3, 3\], \[7, 7\]\], null, \[\[1, 3\], \[7, 7\]\], null, \[\[1, 3\], \[6, 7\]\]\]
 
-**Explanation:** SummaryRanges summaryRanges = new SummaryRanges(); summaryRanges.addNum(1); // arr = \[1\] summaryRanges.getIntervals(); // return \[\[1, 1\]\] summaryRanges.addNum(3); // arr = \[1, 3\] summaryRanges.getIntervals(); // return \[\[1, 1\], \[3, 3\]\] summaryRanges.addNum(7); // arr = \[1, 3, 7\] summaryRanges.getIntervals(); // return \[\[1, 1\], \[3, 3\], \[7, 7\]\] summaryRanges.addNum(2); // arr = \[1, 2, 3, 7\] summaryRanges.getIntervals(); // return \[\[1, 3\], \[7, 7\]\] summaryRanges.addNum(6); // arr = \[1, 2, 3, 6, 7\] summaryRanges.getIntervals(); // return \[\[1, 3\], \[6, 7\]\]
+**Explanation:**
+```
+SummaryRanges summaryRanges = new SummaryRanges(); 
+summaryRanges.addNum(1); // arr = \[1\] 
+summaryRanges.getIntervals(); // return \[\[1, 1\]\] 
+summaryRanges.addNum(3); // arr = \[1, 3\] 
+summaryRanges.getIntervals(); // return \[\[1, 1\], \[3, 3\]\] 
+summaryRanges.addNum(7); // arr = \[1, 3, 7\] 
+summaryRanges.getIntervals(); // return \[\[1, 1\], \[3, 3\], \[7, 7\]\] 
+summaryRanges.addNum(2); // arr = \[1, 2, 3, 7\] 
+summaryRanges.getIntervals(); // return \[\[1, 3\], \[7, 7\]\] 
+summaryRanges.addNum(6); // arr = \[1, 2, 3, 6, 7\] 
+summaryRanges.getIntervals(); // return \[\[1, 3\], \[6, 7\]\]
+```
```

Comment:
Please remove escapes and replace code block with 4 spaces ident.

![image](https://user-images.githubusercontent.com/348496/145738531-f3baee4f-bf90-4efa-a7eb-2ffcbb89846d.png)

Reply:
We may use four space ident for code blocks.